v63004/inref/gtm8923.m

```diff
@@ -0,0 +1,115 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm8923
+
+	SET encodings="UTF-16"
+	SET encodings="UTF-16 UTF-16BE UTF-16LE"
+
+	FOR I=1:1:$L(encodings," ") DO
+	. SET chset=$P(encodings," ",I)
+	. SET file="gtm8923"_chset_".file"
+	.
+	. ;READ * test
+	. OPEN file:CHSET=chset:10
```

Comment:
While having a timeout of 10 seconds in the open command is good, we don't handle the case where the open fails with a timeout. So it is better to remove the :10. If the open command hangs, we will know. But that way we don't need to handle timeout. Same comment applies to multiple places.